README.md

```diff
@@ -30,7 +30,7 @@ You can find information on how to contribute to this project and our docs proje
 - `oc` is connected with adequate permissions to create new namespaces in your OCP cluster.
 - macOS users:
    - `gsed` is required. Install using `brew install gnu-sed`
-   - `wait` is optional. Install using `brew install wait`
+   - `wait` is optional. Install using `brew install juju-wait`
```

Comment:
I think this should be `brew install watch`

Reply:
got it, that makes sense.